hostfactory/install.ps1

```diff
@@ -101,7 +101,7 @@ if (Test-Path -Path "$providerConfPath\\hostProviders.json") {
             "name": "azurecc",
             "enabled": 1,
             "plugin": "azurecc",
-            "confPath": "${HF_CONFDIR}\\providers\\azurecc",
+            "confPath": "${HF_CONFDIR}\\providers\\azurecc\\conf",

```

Comment:
Is this the path used for other providers?   (seems strange that the path would be: .../**conf**/providers/azurecc/**conf**/  )

Reply:
yeah I think we don't need extra conf. For other providers its just /conf/providers/azurecc, I can update it and will update util.py as well